LICENSE.md

```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```

Comment:
The spec file is released under AGPL, here you have GPL which one is correct? I assume this should be changed to AGPL, but it's up to you.

Reply:
All of this is GPL, I fixed that last occurrence of Affero. It is meant to match liquidsoaps license.